### ISSUE TITLE
Missing parameter attribute in a device profile resource operation leads to value descriptors w/out a name

### ISSUE THREAD

**Issue by tonyespy** [edgexfoundry/device-sdk-go#252]
If a device profile includes a device command with a resource operation that lacks a parameter attribute, a value descriptor will be created without a name.

Reproduced using the latest snap from the edge channel version 1.0.0-20190422+6305f79      (804), and the latest device-sdk-go (as of today).

Here's the diff from device-sdk-go:

```
@@ -27,7 +27,7 @@ deviceCommands:
     -
         name: "Switch"
         get:
-            - { operation: "get", object: "SwitchButton", property: "value", parameter: "Switch" }
+            - { operation: "get", object: "SwitchButton", property: "value" }
```

I also removed the AutoEvents defined in the example configuration.toml so that I could remove value descriptors (which appear to immutable if Readings exist that reference them) at will.

Here are the resulting value descriptors created when I run device-simple (you'll notice that the first has no name):

> 
> [
>   {
>     "id": "d2fc4391-3f21-47a5-9c9a-fe7d2be91e84",
>     "created": 1555974857545,
>     "description": "Switch On/Off.",
>     "modified": 1555974857545,
>     "min": null,
>     "max": null,
>     "defaultValue": null,
>     "type": "Bool",
>     "uomLabel": "On/Off",
>     "formatting": "%s"
>   },
>   {
>     "id": "3f9c1050-11c5-4917-8f11-0dfe8ebe4d8c",
>     "created": 1555974857546,
>     "description": "Switch On/Off.",
>     "modified": 1555974857546,
>     "name": "Switch",
>     "min": null,
>     "max": null,
>     "defaultValue": null,
>     "type": "Bool",
>     "uomLabel": "On/Off",
>     "formatting": "%s"
>   },
>   {
>     "id": "2ef4c962-99d6-4ae2-b305-9326b85fb3fa",
>     "created": 1555974857547,
>     "description": "Visual representation of Switch state.",
>     "modified": 1555974857547,
>     "name": "Image",
>     "min": null,
>     "max": null,
>     "defaultValue": null,
>     "type": "Binary",
>     "uomLabel": "On/Off",
>     "formatting": "%s"
>   }
> ]

This is related to issue #150.

**Comment by cloudxxx8** [comment]
This issue should also be fixed by #270

**Comment by cloudxxx8** [comment]
fixed by #270
